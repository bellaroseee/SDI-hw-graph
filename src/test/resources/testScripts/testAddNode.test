# Test that a node can be added to a graph

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add 2 pair of nodes
AddNode graph1 n3
AddNode graph1 n4

# Create another graph
CreateGraph graph2

# Add a pair of nodes
AddNode graph2 n1
AddNode graph2 n2