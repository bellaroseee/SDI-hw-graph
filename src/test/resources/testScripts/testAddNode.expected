# Test that a node can be added to a graph

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add 2 pair of nodes
added node n3 to graph1
added node n4 to graph1

# Create another graph
created graph graph2

# Add a pair of nodes
added node n1 to graph2
added node n2 to graph2